docs/eabbeintu.md

```diff
@@ -0,0 +1,4 @@
+# Bio
+- **Name:** Eli Abbe
+- **Title:** SW Engineer
```

Comment:
What does "SW" stand for? Southwest? [urbandictionary.com](http://www.urbandictionary.com/define.php?term=SW&defid=3199098) has some interesting definitions as well!